src/Components/Publishing/Layouts/NewsLayout.tsx

```diff
@@ -74,8 +84,66 @@ export class NewsLayout extends Component<Props, State> {
     }
   }
 
+  // Ad unit code "Mobile/Desktop_InContentLB1" is for ads that appear after the 3rd article,
+  // "<Mobile/Desktop_InContentLB2" for ads that appear after the 9th article,
+  // and Mobile/Desktop_InContentLBRepeat is for all subsequent ads
```

Comment:
#minor - conventionally multi-line comments above class methods (and functions) are in the dockblock format:

```
/**
 * Ad unit code "Mobile/Desktop_InContentLB1" is for ads that appear after the 
 * 3rd article, "<Mobile/Desktop_InContentLB2" for ads that appear after the 9th 
 * article, and Mobile/Desktop_InContentLBRepeat is for all subsequent ads
 */
```
In VSCode, if you type `/**` and hit enter it should automatically expand. 

That said, it would be better to put the comment above the if branch where the 3 and the 9 live, rather than at the top. People reading the logic inside of the function will appreciate it later.